src/main/java/com/conveyal/datatools/manager/jobs/AutoDeployFeedJob.java

```diff
@@ -76,6 +76,9 @@ public void jobLogic () {
             // First, check to see if there are other fetch/process feed jobs in progress, if there are some still
             // processing (excepting the current feed source), complete job successfully.
             status.completeSuccessfully("Auto-deploy skipped because of feed fetches in progress.");
+            // Set error to true so the 'completeSuccessfully' message is not overwritten by MonitorableJob
+            // to 'Job complete!'.
+            status.error = true;
```

Comment:
Maybe we should just fail this job instead of completing successfully (so that we don't have mismatched status states).

Reply:
@landonreed The unit tests all pass now, so should hopefully work locally for you as well. The failing test needed a unique server or deployment not sure which.